pulsefire/ratelimiters.py

```diff
@@ -73,7 +75,8 @@ async def acquire(self, invocation: Invocation) -> float:
                         "method": invocation.method,
                         "urlformat": invocation.urlformat,
                         "params": invocation.params,
-                    }
+                    },
+                    "limiting_share": self.limiting_share
```

Comment:
I don't think having the client sending the limit coefficient is a good idea, picture this: A sends 0.7, B sends 0.3, the rate limiter will never use more than 0.7 because that is the highest coef it receives. More over this may cause too many point of failures that are hard to debug.

So far the content of this PR supports the following solution which I think is smarter: Create two proxy rate limiters with different limit_perc serving in different ports, then the client uses two different RiotAPIClient pointing to each rate limiter. Would need to write guides on the docs to mention this use-case (I can do it myself in a new commit so dw).

Reply:
I am not sure if I understand this currently.
In this case, I don't see the need for a proxy anymore.
When I create two limiters completely separated from each other, one with 0.3 and one with 0.7.
Or am I missing something?